client/src/redux/root.reducer.ts

```diff
@@ -2,17 +2,20 @@ import { combineReducers } from 'redux';
 import { persistReducer } from 'redux-persist';
 import storage from 'redux-persist/lib/storage';
 
+import { modalReducer } from './modals/modals.reducer';
 import { projectReducer } from './project/project.reducer';
 import { userReducer } from './user/user.reducer';
 
 const persistConfig = {
 	key: 'root',
 	storage,
+	blacklist: ['modals'],
```

Comment:
We don't want to persist modal state, otherwise dropdowns and dialogs would stay open on page reloads